images/cellranger/Dockerfile

```diff
@@ -1,9 +1,18 @@
 FROM ubuntu:20.04
 LABEL maintainer="ccdl@alexslemonade.org"
 
+RUN apt-get update && apt-get install -y build-essential libzip-dev
+
+# Install bcl2fastq
+WORKDIR /tmp
+COPY bcl2fastq2-v2.20.0.422-Source.tar.gz bcl2fastq2.tar.gz
+COPY install-bcl2fastq.sh install-bcl2fastq.sh
+RUN bash install-bcl2fastq.sh
+
```

Comment:
```suggestion
FROM centos:centos8
LABEL maintainer="ccdl@alexslemonade.org"

RUN yum makecache && yum install -y unzip

# Install bcl2fastq
WORKDIR /tmp
COPY bcl2fastq2-v2-20-0-linux-x86-64.zip bcl2fastq2-v2-20-0-linux-x86-64.zip
RUN unzip bcl2fastq2-v2-20-0-linux-x86-64.zip
RUN rpm -i bcl2fastq2-v2.20.0.422-Linux-x86_64.rpm \
&& rm bcl2fastq2-v2.20.0.422-Linux-x86_64.rpm && \
 rm bcl2fastq2-v2-20-0-linux-x86-64.zip
```
I tested this and it builds the image in ~ 2 minutes and I checked by going into the image interactively to make sure that bcl2fastq and cellranger was installed successfully. This isn't necessary since what you had worked, but this way everything is in one file and it's a little simpler.